Persephone/Controllers/AlbumDetailView.swift

```diff
@@ -46,15 +46,15 @@ class AlbumDetailView: NSViewController {
   }
 
   func getAlbumSongs(for album: Album) {
-    App.mpdClient.getAlbumSongs(for: album.mpdAlbum) { (mpdSongs: [MPDClient.MPDSong]) in
-      self.dataSource.setAlbumSongs(
+    App.mpdClient.getAlbumSongs(for: album.mpdAlbum) { [weak self] (mpdSongs: [MPDClient.MPDSong]) in
+      self?.dataSource.setAlbumSongs(
```

Comment:
Maybe `guard let self = self` to ensure that the reference remains valid for the duration of the block?